Gemfile

```diff
@@ -1,98 +1,48 @@
 source 'https://rubygems.org'
 
-ruby '2.4.0'
+ruby '2.6.1'
+gem 'rails', '~> 5.1.6.1'
 
-# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '~> 5.1.3'
 # Use postgresql as the database for Active Record
-gem 'pg', '~> 0.18'
+gem 'pg', '~> 1.1'
 # Use Puma as the app server
-gem 'puma', '~> 3.0'
-# Use SCSS for stylesheets
-gem 'sass-rails', '~> 5.0'
-# Use fontawesome for common icons
-gem 'font-awesome-rails'
-# Use Uglifier as compressor for JavaScript assets
-gem 'uglifier', '>= 1.3.0'
-# Use CoffeeScript for .coffee assets and views
-gem 'coffee-rails', '~> 4.2'
+gem 'puma', '~> 3.12'
 
 gem 'therubyracer', platforms: :ruby
 
-# Use jquery as the JavaScript library
-gem 'jquery-rails'
-# Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
-gem 'turbolinks', '~> 5'
-# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
-gem 'jbuilder', '~> 2.5'
-# Use autoprefixer to avoid writing css prefixes
-gem 'autoprefixer-rails'
-
-gem 'bootstrap-sass'
-
-gem 'carrierwave'
-
 # Authentication
-gem 'devise'
-gem 'devise-async', '~> 0.7.0'
-# devise-i18n support
-gem 'devise-i18n'
-
-gem 'responders'
-
-gem 'active_model_serializers'
-
-gem 'foreman'
+gem 'devise_token_auth', '~> 1.0'
 
-# Active Admin
-gem 'activeadmin', git: 'https://github.com/activeadmin/activeadmin'
-gem 'inherited_resources', git: 'https://github.com/activeadmin/inherited_resources'
-
-# Enables Slim templates
-gem 'slim-rails'
+gem 'active_model_serializers', '~> 0.10'
```

Comment:
Should we considered [Fast Json API](https://github.com/Netflix/fast_jsonapi)? It's really simillar to `ActiveModel::Serializers` and a little more performant.

Reply:
I think it's worth the research but since most Wolox projects use active_model_serializers I'll keep it in this PR

Reply:
For what I've investigated on Fast Json API, it's not exactly the same. It actually has many problems (as in complications, not actual problems) to render nested models which make it more complicated to use.

I'd say that we eventually migrate towards it, but in the meantime we have to stick with ActiveModel Serializers till we fully understand FJAPI

Reply:
As tested in P&G I don't suggest fast json api for now